src/main/java/net/dv8tion/jda/api/requests/restaction/PermissionOverrideAction.java

```diff
@@ -65,23 +63,6 @@
      */
     Member getMember();
 
-    /**
-     * Whether this override is for a {@link Member}
-     *
-     * @return True, if this override is for a member
-     */
-    boolean isMemberOverride();
-
-    /**
-     * Whether this override is for a {@link Role}
-     *
-     * @return True, if this override is for a role
-     */
-    default boolean isRoleOverride()
-    {
-        return !isMemberOverride();
-    }
```

Comment:
These were removed because we already had `isRole` and `isMember` in this interface.